src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -283,6 +306,128 @@ public static void ValidateEntityNamesInConfig(Dictionary<string, Entity> entity
             }
         }
 
+        /// <summary>
+        /// Helper method to validate that the rest path is correctly configured for the entity.
+        /// The rest path can only be a boolean value or a string.
+        /// If configured as a string, it should not be null/empty and should not conflict with the rest path
+        /// configured for any other entity.
+        /// </summary>
+        /// <param name="entityName">Name of the entity.</param>
+        /// <param name="restPathElement">The rest path element for the entity.</param>
```

Comment:
perhaps a rename from `restPathElement` to a more generic name because this object can be a boolean, which does not represent a rest path, it just means rest is enabled/disabled for the entity.

Reply:
But even in that case,  `"path": true`. So, either we change the feature name in config and here or this will be fine as is. I have mentioned in the comments as well, what are the expected value types the restPath can take.

Reply:
Not relevant anymore as I misinterpreted that the rest path can be a boolean. It can only be a string.